src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -746,6 +746,14 @@ export const needsToAnswerUnemployability = formData =>
   needsToEnterUnemployability(formData) &&
   _.get('view:unemployabilityUploadChoice', formData, '') === 'answerQuestions';
 
+export const doctorsCare = formData =>
```

Comment:
better names, something like `hasDoctorsCare` to indicate it's returning a boolean

Reply:
Also unit tests

Reply:
though the format seems to be `needs`